arm-customer-insights/2016-01-01/examples/ProfilesListByHub.json

```diff
@@ -0,0 +1,373 @@
+{
+	"parameters": {
+		"hubName": "sdkTestHub",
+		"resourceGroupName": "TestHubRG",
+		"api-version": "2016-01-01",
+		"subscriptionId": "subid"
+	},
+	"responses": {
+		"200": {
+			"body": {
+				[{
+					"id": "/subscriptions/c909e979-ef71-4def-a970-bc7c154db8c5/resourceGroups/TestHubRG/providers/Microsoft.CustomerInsights/hubs/azSdkTestHub/profiles/TestProfileType18055",
```

Comment:
the array is the value of the "value" property as per the swagger spec.

General comment: The body can be one of object, array, string, boolean, number, null. So please provide the example of response body, directly inside the curly braces "body" property.